src/main/java/frc/robot/subsystems/ArmSubsystem.kt

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems
+
+import com.revrobotics.CANSparkMax
+import com.revrobotics.SparkMaxAbsoluteEncoder
+import com.revrobotics.SparkMaxLimitSwitch
+import edu.wpi.first.networktables.NetworkTableInstance
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+import frc.robot.ArmConstants
+
+class ArmSubsystem(val elbowMotor: CANSparkMax, val elevatorMotor: CANSparkMax) :SubsystemBase() {
```

Comment:
Also, for this year, instead of passing in motor objects to classes, we should pass in CAN ids. In this case, change elbowmotor and elevatormotor from being of type CANSparkMax to being of type Int.